cdap-docs/developer-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -687,13 +687,44 @@ CDAP pipelines as of version |version|.
             }
           }
 
+   * - ``toggle-switch``
+     - - ``on``: 'On' state of the switch, defined by
```

Comment:
remove the extra `-`

Reply:
This extra `-` is to show the attributes (`on`, `off`, and `default`) as bullet points inside the table cell.